apps/root/src/common/components/contact-modal/contact-item/index.tsx

```diff
@@ -56,8 +56,7 @@ const StyledContactItem = styled(Grid)<{ menuOpen: boolean }>`
 `}
 `;
 
-const StyledContactLabel = styled(Typography).attrs({ variant: 'h6' })`
-  font-weight: bold;
+const StyledContactLabel = styled(Typography).attrs({ variant: 'bodyBold' })`
```

Comment:
<img width="764" alt="Screenshot 2024-09-25 at 12 42 53 am" src="https://github.com/user-attachments/assets/608a3a3f-f1cb-44b8-a8eb-5575525cde59">


This should be h6 right?

Reply:
As h6 is deprecated, I based in the css props, that match with the bodyBold variant